src/main/java/com/google/devtools/build/lib/packages/RepoCallable.java

```diff
@@ -27,6 +29,25 @@ private RepoCallable() {}
 
   public static final RepoCallable INSTANCE = new RepoCallable();
 
+  @StarlarkMethod(
+      name = "ignore_directories",
+      useStarlarkThread = true,
+      documented = false, // TODO
+      parameters = {
+        @Param(
+            name = "dirs",
+            allowedTypes = {
+              @ParamType(type = Sequence.class, generic1 = String.class),
+            })
+      })
+  public Object ignoreDirectories(Iterable<?> dirsUnchecked, StarlarkThread thread)
```

Comment:
does just returning void not work?

Reply:
Indeed, I just copy-pasted it from `repoCallable()`. It works alright.